src/farcasterd/runtime.rs

```diff
@@ -2080,28 +1149,7 @@ fn launch_swapd(
     )?;
     let msg = format!("New instance of swapd launched with PID {}", child.id());
     debug!("{}", msg);
```

Comment:
great that these are now minimized (of course, still pending a potential merge of swap id & offer uuids)